.github/workflows/verify.yaml

```diff
@@ -85,7 +85,8 @@ jobs:
     name: Kubernetes Tests
     runs-on: ubuntu-18.04
     needs: ["build-verify"]
-    if: github.repository_owner == 'Apicurio' && github.event_name == 'push' && github.ref == 'refs/heads/master'
+    # if: github.repository_owner == 'Apicurio' && github.event_name == 'push' && github.ref == 'refs/heads/master'
+    if: github.repository_owner == 'Apicurio'
```

Comment:
Should we keep this so that the k8s tests run for all PRs?

Reply:
Yes, I think that is a good idea to keep this. @famartinrh thoughts?

Reply:
yes please!